src/tokens/hooks.ts

```diff
@@ -61,6 +67,51 @@ export function useTokenPricesAreStale(networkIds: NetworkId[]) {
   }
 }
 
+export function useSendableTokens() {
+  // TODO: get this from statsig
```

Comment:
is there an issue where this will be done? or is it something you plan to do in a followup PR?

Reply:
todo addressed after #4265 was merged